modules.yml

```diff
@@ -851,16 +851,17 @@
   - binaryedge_api
   version: '1.0'
 - author: Tim Tomes (@lanmaster53) and Matt Puckett (@t3lc0)
-  dependencies: []
+  dependencies:
+    - shodan
   description: Harvests port information from the Shodan API by using the 'ip' search
     operator. Updates the 'ports' table with the results.
   files: []
-  last_updated: '2019-06-24'
+  last_updated: '2020-06-21'
   name: Shodan IP Enumerator
   path: recon/hosts-ports/shodan_ip
   required_keys:
   - shodan_api
-  version: '1.0'
+  version: '1.2'
```

Comment:
Why the version hop from 1.0 to 1.2? What happened to 1.1?

Reply:
I did it in two different steps and it was muscle memory taking over.